### PR TITLE
Guarantee mining shelter egress

### LINKED ISSUE #26085: Bluespace shelter capsules are impossible to escape from with Protokinetic Crushers
Server Revision: 8d1ee2267ad7231824b725baffd96daa643cdfc2 

If you only have the protokinetic crusher and you activate a bluespace shelter capsule, you can become trapped if the shelter's airlock is against rock like this.  There is air in the shelter, so the crusher will click and be unable to mine the rock.

issue

### PR DESCRIPTION
**Pull request by octareenroon91**
Fixes #26085

:cl:
add: Bluespace capsule now include a pocket pickaxe, in case you lost yours.
/:cl:

I also added a stack of cloth for bags or burnt jumpsuits.

### CODE REVIEW

**Comment by Cyberboss** [comment]
This is a tweak, fyi

**Comment by octareenroon91** [comment]
Just offering a PR to match @Dorsisdwarf 's comment

**Comment by octareenroon91** [comment]
Also this dances on the edge of being a mapping change.

**Comment by FantasticFwoosh** [comment]
The only people to have pod capsules by design would already be miners, so i think this is unnessecary when you could just craft a bone pickaxe or salvage/print a pickaxe from a autolathe.

> Downvote from me on powercreep tweaks during a freeze & because i oppose the idea.

**Comment by octareenroon91** [comment]
You have to be pretty perverse or unlucky to get yourself trapped, it's true. Blame the crusher's mining inefficiency.

Opinions on just running pick() out to the template perimeter?

**Comment by ExcessiveUseOfCobblestone** [comment]
Stop using the hardmode item if you want to be handheld jesus.

Also disgusting """""QoL"""""" change mixed with the "fix".

**Comment by Repukan** [comment]
Stop with this bullshit.

Just grab the one you get roundstart.

Trying to QoL your fuck ups out of the game makes a boring experience

**Comment by Repukan** [comment]
Also this isn't fixing anything because there was nothing inherently broken in the first place.

And why the fuck are you adding cloth when you get a bedsheet?

**Comment by BeeSting12** [comment]
why would you not carry a PKA on you anyways?

**Comment by Dorsisdwarf** [comment]
If you're carrying the crusher you wont usually carry a PKA too because its big and your slots are quite limited.

But yes, this isnt a fix and therefore can go away right now (unless the freeze ended in the last  day and I didnt get the memo)

File: code/modules/mining/mine_items.dm [hunk 1]
```diff
@@ -404,6 +404,8 @@
 	for(var/i in 1 to 5)
 		var/obj/item/weapon/reagent_containers/food/snacks/donkpocket/warm/W = new(src)
 		load(W)
+	var/obj/item/weapon/pickaxe/mini/P = new(src)
+	var/obj/item/stack/sheet/cloth/ten/C = new(src)
```
Comment (francinum) [inline code/modules/mining/mine_items.dm]: Defined but lot loaded, this is why travis is screaming at you.

**Review by francinum — CHANGES_REQUESTED** [review]

**Comment by optimumtact** [comment]
freeze

**Comment by octareenroon91** [comment]
Thanks but when can you close the issue?

**Pull request closed without merging**